text_extensions_for_pandas/io/bert.py

```diff
@@ -118,14 +118,18 @@ def make_bert_tokens(target_text: str, tokenizer) -> pd.DataFrame:
     return token_features
 
 
-def add_embeddings(df: pd.DataFrame, bert: Any) -> pd.DataFrame:
+def add_embeddings(df: pd.DataFrame, bert: Any,
+                   overlap: int = 32, non_overlap: int = 64) -> pd.DataFrame:
     """
     Add BERT embeddings to a DataFrame of BERT tokens.
 
     :param df: DataFrame containing BERT tokens, as returned by
       :func:`make_bert_tokens` Must contain a column
       `input_id` containing token IDs.
     :param bert: PyTorch-based BERT model from the `transformers` library
+    :param overlap: (optional) how much overlap there should be between adjacent windows
+    :param non_overlap: (optional) how much non-overlapping content between the
+     overlapping regions there should be at the middle of each window?
```

Comment:
is the `non_overlap` param a rhetorical question? haha j/k
also the one below too

Reply:
Yeah, window parameters are confusing. We should probably put an illustration into the docs at some point.

Reply:
Well, I was just referring to the question mark at the end of the description..